services/src/main/java/org/keycloak/authentication/ParResponse.java

```diff
@@ -0,0 +1,28 @@
+package org.keycloak.authentication;
```

Comment:
It might be preferable to put PAR related classes on to its dedicated package. (ex. org.keycloak.protocol.par)

Reply:
Done, thank you.

Reply:
I've confirmed it. Thank you.